docs/index.rst

```diff
@@ -6,6 +6,12 @@
 ctwrap - Python Wrapper for Batch Simulations
 =============================================
 
+.. include:: ../README.rst
+
+.. include:: ../install.rst
+
+.. include:: ../CONTRIBUTING.rst
+
 .. toctree::
    :maxdepth: 2
    :caption: Contents:
```

Comment:
Please leave the table of contents at the top of the page (it is very useful for navigation).